test/migrations-test.coffee

```diff
@@ -150,7 +150,7 @@ describe "Migrations",  ->
     describe "v-1.23.0 changes", ->
       describe "a known failing case", ->
         # This test case is based on a field-reported bug whereby a simulation
-        # containg collector nodes was not coreectly migrating the simulationType
+        # containg collector nodes was not corectly migrating the simulationType
```

Comment:
Close enough... :)

`containg` => `containing` collector nodes was not `corectly` => `correctly` migrating the simulationType